Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/FormHeader.swift

```diff
@@ -22,7 +22,12 @@ struct FormHeader: View {
     var body: some View {
         if let title {
             Text(title)
-                .font(.largeTitle)
+                .font(.title)
+                .fontWeight(.bold)
+            // Having this padding on `FormHeader`, caused padding on both
+            // the top AND bottom of the header. Odd...
+                .padding(.top, 16)
+            Divider()
```

Comment:
I agree about it being more symmetrical. I made the change, but since the `Divider()` was only shown if there was a title, I had to move that check into the `FeatureFormView.body` as well. I could go either way, so let me know what you prefer.

Reply:
I think that works nicely, thanks!